input/otlp/logs.go

```diff
@@ -109,15 +110,54 @@ func (c *Consumer) convertLogRecord(
 		}
 		event.Span.ID = spanID.HexString()
 	}
-	if attrs := record.Attributes(); attrs.Len() > 0 {
-		setLabels(attrs, &event)
+	attrs := record.Attributes()
+	setLabels(attrs, &event)
+	if event.Error != nil {
+		event.Processor = model.ErrorProcessor
+		event.Event.Kind = "event"
+		event.Event.Type = "error"
 	}
 	return event
 }
 
 func setLabels(m pcommon.Map, event *model.APMEvent) {
+	var exceptionEscaped bool
+	var exceptionMessage, exceptionStacktrace, exceptionType string
 	m.Range(func(k string, v pcommon.Value) bool {
-		setLabel(k, event, ifaceAttributeValue(v))
+		switch k {
+		case semconv.AttributeExceptionMessage:
+			exceptionMessage = v.Str()
+		case semconv.AttributeExceptionStacktrace:
+			exceptionStacktrace = v.Str()
+		case semconv.AttributeExceptionType:
+			exceptionType = v.Str()
+		case semconv.AttributeExceptionEscaped:
+			exceptionEscaped = v.Bool()
+		default:
+			setLabel(replaceDots(k), event, ifaceAttributeValue(v))
```

Comment:
There seems to be a few inconsistencies with the usage of `replaceDots` and `setLabel` in the code. What's the correct behaviour ?

Reply:
We should be consistently replacing dots.